doc/modules/model_evaluation.rst

```diff
@@ -2317,10 +2355,16 @@ function::
     for an example of mean squared error usage to
     evaluate gradient boosting regression.
 
+|details-start|
+**Square root of the MSE**
+|details-split|
+
 Taking the square root of the MSE, called the root mean squared error (RMSE), is another
 common metric that provides a measure in the same units as the target variable. RSME is
 available through the :func:`root_mean_squared_error` function.
 
+|details-end|
+
```

Comment:
This paragraph is short enough not to be hidden in a dropdown. It is also a common use metric, so we rather keep it visible.

Reply:
Very good! It has been de-dropdowned in one of the following commits.